### PR TITLE
250 indp notebook

### PR DESCRIPTION
**Pull request by longshuicy**
Note: There is an endpoint implemented in dev but not prod yet. This will cause the building damage to fail. To continue testing INDP, a workaround cell is added **(BUT before merging into develop this cell needs to go away)**. To test, skip the building damage cell for now and run the temporary cell instead

image

----------------------

- Install https://github.com/IN-CORE/pyincore/pull/366 this branch. If you already have a conda environment, i just do `pip install . -U --no-dependencies` to cut some corner
- Install SCIP if it's not there. I use mamba `mamba install scip`

### CODE REVIEW

**Comment by ylyangtw** [comment]
I tested it locally but I am not sure why it is accessing the prod... And the feature is not in prod yet, so the notebook stops.
image

**Comment by ywkim312** [comment]
> I tested it locally but I am not sure why it is accessing the prod... And the feature is not in prod yet, so the notebook stops. image

if you want to point incore-dev please change your client like this 
client = IncoreClient("https://incore-dev.ncsa.illinois.edu")

**Comment by ylyangtw** [comment]
Notebook works
image

**Review by ylyangtw — APPROVED** [review]
Changes look good~ Approve

File: notebooks/indp.ipynb [hunk 1]
```diff
@@ -0,0 +1,724 @@
...
+    "epf_rest.run_analysis()\n",
+    "epf_restoration_time = epf_rest.get_output_dataset(\"repair_times\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a467d0d1",
+   "metadata": {},
+   "source": [
+    "### Eletric Power Facility Monte Carlos Simulation"
```
Comment (navarroc) [inline notebooks/indp.ipynb:269]: Electric

File: notebooks/indp.ipynb [hunk 2]
```diff
@@ -0,0 +1,724 @@
...
+    "epf_sample_damage_states = epf_mc.get_output_dataset(\"sample_damage_states\")\n",
+    "epf_sample_damage_states.get_dataframe_from_csv().head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a4b91790",
+   "metadata": {},
+   "source": [
+    "### Eletric Power Facility Repair Cost"
```
Comment (navarroc) [inline notebooks/indp.ipynb:297]: Same, should be Electric

**Review by navarroc — APPROVED** [review]

**Pull request merged**